src/actions.js

```diff
@@ -418,7 +466,7 @@ export const fetchNewTxTokenBalance = async (wallet, tx) => {
     return null;
   }
   const updatedBalanceMap = {};
-  const balances = wallet.getTxBalance(tx);
+  const balances = await wallet.getTxBalance(tx);
```

Comment:
The old facade `getTxBalance` method is not `async`, will this continue working for the old facade?

Reply:
Yes it will, just tested it